### ISSUE TITLE
Principles of DevOps: Introduction

### ISSUE THREAD

**Issue by garritfra** [garritfra/garrit.xyz#311]
I recently changed roles in my company, and I can officially call myself a "DevOps Engineer" now. But what does that really mean?

In an attempt to write down my thoughts about this topic, I'm starting a series of blog posts called "Principles of DevOps". I'm usually very bad at sticking to things, so I'm curious to see if this series will lead anywhere.

To collect the posts of this series, I created a tag called [#PrinciplesOfDevOps](/posts?tags=PrinciplesOfDevOps). If you're reading this in the future, be sure to check out this tag to see all installments.

## What is DevOps?

Let's kick off the series with a very basic question: What on earth is DevOps?

DevOps is often as an inflationary term to describe "whatever comes after dev". This can't be further from the truth.

In the past, developers, operations, designers, QA and other stakeholders of an applications were often implicitly trained to work in "silos". Once designers have finished their job, they pass their mockups to developers. When developers are done writing the application, they pass their code to operations, whose job it is to deploy it.

DevOps is a set of practices that aims to combine the work of project stakeholders to unite people, process, and technology in application planning, development, delivery, and operations. Although the term DevOps only consists of "Dev" and "Ops", it has since evolved to include design, quality assurance and security. You may have heard of "DevSecOps", which aims to incorporate more roles into the term, but "DevOps" seems to stick the best with most people.

## What does a "DevOps Engineer" do?

I recently wrote a blog post about this: [The role of a DevOps Engineer](/posts/2023-04-22-the-role-of-a-devops-engineer).

In short, the job of a DevOps Engineer is to reduce the friction between stakeholders of a project. A collegue of explained this in a really good way:

> A DevOps Engineer doesn't push the button, they enable the developers to push the button themselves.

## Let's jump in!

I hope by now you have a vague sense of what DevOps is. Next up, I want to uncover the principles and practices of DevOps. Thanks for reading to the end!

---

This is post 072 of [#100DaysToOffload](https://100daystooffload.com/).
